project-base/app/config/routes/shopsys_front.yaml

```diff
@@ -11,5 +11,12 @@ front_export_personal_data:
         _controller: App\Controller\Front\PersonalDataController::exportXmlAction
         type: ~
     path: /personal-overview-export/xml/{hash}
+
 front_homepage:
     path: /
+
+front_social_network_login:
+    defaults: { _controller: App\Controller\Front\SocialNetworkController::loginAction }
+    path: /social-network/login/{type}
+    requirements:
+        type: Google|Facebook|Seznam #copy from /packages/framework/src/Model/SocialNetwork/SocialNetworkConfigFactory.php
```

Comment:
is it necessary to check the type on this level? You would have to think about this place when implementing a new type

Reply:
fixed in https://github.com/shopsys/shopsys/pull/3154/commits/68ae66fd9ef6fd6cfcdaf20831bd36b8e38f3186